### PR TITLE
Remove Bisq

### PR DESCRIPTION
**Pull request by ShmoogleOsukami**
This removed Bisq from the website.

As far as I know it was in 3 locations, the header, footer and _acquire_details.

### CODE REVIEW

File: source/layouts/partials/_header.erb [hunk 1]
```diff
@@ -80,10 +80,6 @@
                     </a>
                     <div class="dropdown-menu" aria-labelledby="navbarDropdown">
                       <a class="dropdown-header">Exchanges</a>
-                        <!--
-                          <a class="dropdown-item" href="https://bitshares.openledger.info/#/market/OPEN.GRC_OPEN.BTC"><i class="fa fa-exchange"></i> OpenLedger</a>
```
Comment (barton2526) [inline source/layouts/partials/_header.erb]: Can we keep this part commented out?  It is feasible that OpenLedger will be usable again if they ever fix their gateway.

**Comment by barton2526** [comment]
@ShmoogleOsukami Can we also remove the image assets?  https://github.com/gridcoin-community/Gridcoin-Site/blob/master/source/assets/img/Exchanges/bisq.png

File: source/layouts/partials/_header.erb [hunk 2]
```diff
@@ -80,6 +80,10 @@
                     </a>
                     <div class="dropdown-menu" aria-labelledby="navbarDropdown">
                       <a class="dropdown-header">Exchanges</a>
+                        <!--
+                          <a class="dropdown-item" href="https://bitshares.openledger.info/#/market/OPEN.GRC_OPEN.BTC"><i class="fa fa-exchange"></i> OpenLedger</a>
+                        -->
+                        <a class="dropdown-item" href="https://bisq.network/">bisq</a>
```
Comment (barton2526) [inline source/layouts/partials/_header.erb]: still need to remove this line (86)

**Comment by ShmoogleOsukami** [comment]
Think I got everything this time. :D

**Comment by barton2526** [comment]
Approved

**Pull request merged**